tests/test-values.yaml

```diff
@@ -0,0 +1,8 @@
+provider:
+  kind: azure
+kubernetes:
+  caPem: caPem
+clusterID: test1
+baseDomain: test1.example.io
+image:
+  tag: 1.5.4-1d727745faced189044fb83f9736ada29f9e9e12
```

Comment:
I had to add this because otherwise `ats` would try to pull an image tagged `[[ .Version ]]` which does not exist.
If I set the version specifically in the chart (1.5.4, same as `appVersion`) we also get a problem here since the actual app has changed since then (disabling RUM is possible).
This needs to be done after the first release as well.